pkg/forth/forth.go

```diff
@@ -0,0 +1,274 @@
+// the forth package is designed for use by programs
```

Comment:
Add license

Reply:
well, that's tricky. There was no license originally. I guess since I'm a u-root author I can add that license.

Reply:
done